src/integrations.ts

```diff
@@ -0,0 +1,330 @@
+export type Tag = "CMS" | "E-commerce" | "Library" | "Mobile";
+
+export interface Integration {
+  name: string;
+  slug: string;
+  tags: Tag[];
+  fcVersion: "v1" | "v2";
```

Comment:
Should this maybe just be an array for integrations that support both versions? I'm guessing this will become more common over time.

```suggestion
  fcVersions: ("v1" | "v2")[];
```